charts/enterprise-kyverno-operator/templates/cr-custompolicysets.yaml

```diff
@@ -0,0 +1,21 @@
+{{- range .Values.policies.customPolicySetCharts }}
+---
+apiVersion: security.nirmata.io/v1alpha1
+kind: PolicySet
+metadata:
+  annotations:
+  labels:
+    app.kubernetes.io/name: policyset
+    app.kubernetes.io/instance: {{ .name }}
+    app.kubernetes.io/part-of: nirmata-kyverno-operator
+  name: {{ .name }}
+  namespace: {{ template "enterprise-kyverno.namespace" $ }}
+spec:
+  source:
+    helm:
+      chartRepo: {{ .chartRepo }}
+      chartName: {{ .chartName }}
+      version: {{ .version }}
+      username: {{ .username }}
+      passwordSecret: {{ .passwordSecret }}
+  {{- end }}
```

Comment:
Add an empty line, github is flagging as red (ref: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3)